### ISSUE TITLE
[New App Shortcut]: Kinopub

### ISSUE THREAD

**Issue by Zakhse** [PRProd/HA-Firemote#283]
### App Name

Kinopub

### Device Platform

Xiaomi

### App Store Link and/or URL to .apk file if sideloaded

https://kpdl.cc/k.apk

### Application Details

```
adb shell am start -n com.kinopub/com.kinopub.activity.LaunchActivity
```

### Optional Comments

Streaming service https://kino.pub
FAQ: https://kpdl.cc/faq.html

**Comment by PRProd** [comment]
Greetings @Zakhse!

Since I am in a development cycle for [Roku support](https://github.com/PRProd/HA-Firemote/discussions/275) this is not yet available in the official Firemote latest release.  However, if you'd like to use it now, it is available in the latest beta version [V3.2.0b3](https://github.com/PRProd/HA-Firemote/releases/tag/v3.2.0b3) which I have just published.

If you'd like, I can let you know when this has been published into the latest official release.

-Doug

**Comment by PRProd** [comment]
@Zakhse ~

This has been pushed to the official Firemote release version [v3.2.0](https://github.com/PRProd/HA-Firemote/releases/tag/v3.2.0).  Enjoy!
